test/bridge.test.ts

```diff
@@ -5,6 +5,7 @@ import { expectNoClientErrors, parseData, renderPage } from './utils'
 
 const isWebpack = process.env.TEST_BUILDER === 'webpack'
 const isDev = process.env.TEST_ENV === 'dev'
+const isTSX = process.env.TEST_TYPESCRIPT === 'isTSX' || process.env.TEST_TYPESCRIPT !== 'esbuild'
```

Comment:
do we not expect TSX to work with `esbuild`?

Or maybe we should always set `isTSX` and only use matrix for testing esbuild enabled/not?